build.zig.zon

```diff
@@ -0,0 +1,51 @@
+.{
+    .name = "neovim",
+    .version = "0.0.0",
+
+    // This field is optional.
+    // This is currently advisory only; Zig does not yet do anything
+    // with this value.
+    //.minimum_zig_version = "0.12.0",
+
+    // lpeg fetching from https://www.inf.puc-rio.br/~roberto/lpeg/lpeg-1.0.2.tar.gz doesn't work, bundlevendor for now
+
+    .dependencies = .{
+        .ziglua = .{
+            .url = "https://github.com/natecraddock/ziglua/archive/5a7269eaddb13b118818690cea613ab25803683d.tar.gz",
```

Comment:
I tried this PR just of curiosity, and had issues to reproduce, which is mainly because archive URLs are used and they are not guaranteed to have the same hash everytime (see https://github.com/orgs/community/discussions/46034)

If we could switch to git commit hashes instead e.g:
```
zig fetch --save git+https://github.com/natecraddock/ziglua#5a7269eaddb13b118818690cea613ab25803683d
```

this should solve the issue.

Reply:
Great suggestion. I presume this also applies to tag based archives ?

Reply:
yes - the hashes are only stable if the author uploads their own tarballs on github releases

Reply:
> Great suggestion. I presume this also applies to tag based archives ?

If I understood it correctly yes, because they rely on `git archive` command, and create it on the fly each time you request it and only cache it for a while to reduce the workload. And `git archive` does not guarantee that reproducibility.